packages/dx-scheduler-core/src/utils.ts

```diff
@@ -89,14 +89,17 @@ export const getAppointmentStyle: PureComputed<
 > = ({
   top, left,
   width, height,
-}) => ({
-  height,
-  width: `${width}%`,
-  transform: `translateY(${top}px)`,
-  msTransform: `translateY(${top}px)`,
-  left: `${left}%`,
-  position: 'absolute',
-});
+}) => {
+  const transform = `translateY(${top}px)` as any;
```

Comment:
Why do you use `any` here? Could you replace it with the `string` type?

Reply:
No i can't
![image](https://github.com/DevExpress/devextreme-reactive/assets/27704588/9998d4ff-a087-47d9-bcac-4280f029db2c)